### DISCUSSION TITLE
cudaGetDevice

### DISCUSSION THREAD

**Discussion by albrono** [deepmodeling/deepmd-kit#1170] [Q&A]
Hello,
I have installed the offline package of deepmd v2.0.0 (CUDA10.1) on an HCP, but I get the below error when I run the "dp train" command:
RuntimeError: cudaGetDevice() failed. Status: CUDA driver version is insufficient for CUDA runtime version
The GPU is:
TITAN X (Pascal), computeCapability: 6.1
The Cuda version is:
Cuda compilation tools, release 10.0, V10.0.130
How can I solve the issue? ("I cannot update the CUDA driver on the cluster")
Thanks,

**Comment by amcadmus** [comment]
just as the error message said " CUDA driver version is insufficient for CUDA runtime version ". plz upgrade the driver to match the cuda runtime
